src/cloudflare/internal/test/autorag/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+load("//:build/wd_test.bzl", "wd_test")
+load("//src/workerd/server/tests/python:py_wd_test.bzl", "py_wd_test")
+
+wd_test(
+    src = "autorag-api-test.wd-test",
+    args = ["--experimental"],
+    data = glob(["*.js"]),
+)
+
+py_wd_test(
+    size = "large",
+    src = "python-autorag-api-test.wd-test",
+    args = ["--experimental"],
+    data = glob([
+        "*.js",
+        "*.py",
+    ]),
+    # Works but times out
+    make_snapshot = False,
+    tags = [
+        # TODO(someday): Fix asan failure for this, see https://github.com/cloudflare/workerd/pull/3140#discussion_r1858273318
```

Comment:
looks like all (many?) pyodide tests are not asan-compatible? Do you mind creating an issue instead? thanks.

Reply:
i don't have much insight over what is compatible or not with pyodide, i just followed what other python tests are doing